routes/note.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+const router = express.Router();
+const { Note } = require('./db');
+
+function create(req, res, next) {
+   return Note.create(req.body)
+    .then(note => res.send(note))
+    .catch(err => next(err));
+  }
+
+function list(req, res, next) {
+  return Note.find()
+    .then(notes => res.send(notes))
+    .catch(err => next(err));
+};
+
+function find(req, res, next) {
+  const id = req.params.id;
+
+  return Note.findById(id)
+    .then(note => res.send(note))
+    .catch(err => next(err));
+};
+
+function update(req, res, next) {
+  return Note.findByIdAndUpdate(req.params.id, req.body)
+    .then(note => res.send(note))
+    .catch(err => next(err));
+};
+
+function remove(req, res, next) {
+  return Note.findByIdAndRemove(req.params.id)
+    .catch(err => next(err));
```

Comment:
You are missing `res.send` here. This way request will timeout since the server will never send a response.